server/schema.ts

```diff
@@ -399,3 +399,21 @@ export interface HackGTMetrics {
 	values: object;
 	hackgtmetricsversion: number;
 }
+
+export interface IHelpScoutEmailNotFoundTemplate {
+	email: string;
+}
+
+export interface IHelpScoutMainTemplate {
+	name: string;
+	email: string;
+	questions?: IFormItem[];
+	applied: boolean;
+	accepted: boolean;
+	confirmed: boolean;
+	applicationSubmitTime: string | undefined;
```

Comment:
All instances of `key: string | undefined;` can be abbreviated to `key?: string;`